clients/fides-js/src/lib/cookie.ts

```diff
@@ -260,6 +260,11 @@ export const updateExperienceFromCookieConsentNotices = ({
   cookie: FidesCookie;
   debug?: boolean;
 }): PrivacyExperience => {
+  // If the given experience has no notices, return immediately and do not mutate
+  // the experience object in any way
+  if (!experience.privacy_notices) {
+    return experience;
+  }
```

Comment:
This was an unrelated, but subtle bug. I found that if there was a saved consent cookie and a prefetched empty experience (e.g. `experience: {}`), this function would get called and edit the experience to become `{ privacy_notices: undefined }`, which is unexpected by the rest of the code base.

The impact of this is probably zero-to-none, it'd be a really extreme edge case where a user (1) visits a website and gets a valid experience, (2) saves their consent preference, (3) visits the website again from a different location without an experience, (4) this method causes the experience to appear non-empty but without notices, (5) client will then refetch the experience client-side, slowing down the init.

Just fixing this up.